portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -113,7 +113,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         String name = (String) userInfoEntity.getProperty("name");
 
         // If the user opted to post the comment anonymously, make the name Anonymous.
-        String anonymous = getParameter(request, "anonymous", "off").orElse(null);
+        String anonymous = getParameter(request, "anonymous", "off").orElse("off");
```

Comment:
I don't think you need a defaultValue argument in getParameter because the orElse() should handle that.

Reply:
True! I will create a different branch to deal with this issue because all of my getParameter function calls already have a default parameter set so there are a lot of them that are redundant and will need to have that adjusted.